js/src/views/Application/Requests/savedRequests.spec.js

```diff
@@ -19,27 +19,37 @@ import store from 'store';
 
 import SavedRequests, { LS_REQUESTS_KEY } from './savedRequests';
 
+const NETWORK_ID = 42;
 const DEFAULT_REQUEST = {
   requestId: '0x1',
   transaction: {}
 };
 
-const api = createApi();
+const api = createApi(NETWORK_ID);
+const api2 = createApi(1);
 const savedRequests = new SavedRequests();
 
-function createApi () {
+function createApi (networkVersion) {
   return {
     parity: {
       checkRequest: sinon.stub().resolves()
+    },
+    net: {
+      version: sinon.stub().resolves(networkVersion)
     }
   };
 }
 
 describe('views/Application/Requests/savedRequests', () => {
-  beforeEach(() => {
+  beforeEach((done) => {
     store.set(LS_REQUESTS_KEY, {
-      [DEFAULT_REQUEST.requestId]: DEFAULT_REQUEST
+      [NETWORK_ID]: {
+        [DEFAULT_REQUEST.requestId]: DEFAULT_REQUEST
+      }
     });
+
+    savedRequests.load(api)
```

Comment:
Could just return the Promise here instead of calling done. (Mocha is promise-aware)